CrowsZA/functions/fn_spawnIEDClutter.sqf

```diff
@@ -154,9 +210,23 @@ for "_i" from 1 to _iedAmount do {
 	};
 
 	private _ied = selectRandom _ieds;
-	_ied = _ied createVehicle _safePos;
-	_ied setDir (random 360);
 
-	// TODO: This does not work for (armed) explosives
-	//["zen_common_addObjects", [[_ied], objNull]] call CBA_fnc_serverEvent;
+	// If ied type is set to clutter (or random, with a 0.33 chance)
+	// place an extra piece of clutter, and hide an invisible ied beneath it;
+	// it should still be able to be detonated and defused as normal
+	if(_iedType == 2 || (_iedType == 3 && (random 1) < 0.33)) then {
+		_clutter = (selectRandom _smallClutter) createVehicle _safePos;
+		_clutter setDir (random 360);
+		_clutter enableSimulationGlobal false;
+		["zen_common_addObjects", [[_clutter], objNull]] call CBA_fnc_serverEvent;
+
+		_ied = createVehicle [_ied, _safePos, [], 0, "CAN_COLLIDE"];
+		hideObjectGlobal _ied;
```

Comment:
If someone disarms it, and picks it up. How does the hideObject react? Does it appear again when disarmed,  or will it be like an invisible object you can carry around?

Reply:
It just reappears (I think because it spawns a totally new object)

Reply:
perfect. Sounds good!